recipes/breakpad/all/conanfile.py

```diff
@@ -0,0 +1,118 @@
+        )
+        tools.replace_in_file(os.path.join(self._source_subfolder, "Makefile.in"),
+                            "install-includelssHEADERS install-includepHEADERS ",
+                            "iinstall-includepHEADERS "
+        )
+        files_to_patch = [
+            "src/tools/linux/md2core/minidump-2-core.cc",
+            "src/processor/testdata/linux_test_app.cc",
+            "src/common/memory_allocator.h",
+            "src/common/linux/memory_mapped_file.cc",
+            "src/common/linux/file_id.cc",
+            "src/common/linux/safe_readlink.cc",
+            "src/client/minidump_file_writer.cc",
+            "src/client/linux/handler/exception_handler.cc",
+            "src/client/linux/handler/exception_handler_unittest.cc",
+            "src/client/linux/log/log.cc",
+            "src/client/linux/crash_generation/crash_generation_client.cc",
+            "src/client/linux/minidump_writer/linux_dumper.cc",
+            "src/client/linux/minidump_writer/linux_dumper_unittest_helper.cc",
+            "src/client/linux/minidump_writer/proc_cpuinfo_reader.h",
+            "src/client/linux/minidump_writer/minidump_writer.cc",
+            "src/client/linux/minidump_writer/linux_ptrace_dumper.cc",
+            "src/client/linux/minidump_writer/cpu_set.h",
+            "src/client/linux/minidump_writer/directory_reader.h",
+            "src/client/linux/minidump_writer/line_reader.h"
+        ]
+
+        for file in files_to_patch:
+            tools.replace_in_file(
+                os.path.join(self._source_subfolder, file),
+                "#include \"third_party/lss/linux_syscall_support.h\"",
+                "#include <linux_syscall_support.h>"
+            )
+
+        # Let Conan handle fPIC
+        tools.replace_in_file(
+            os.path.join(self._source_subfolder, "Makefile.in"),
+            textwrap.dedent("""\
+                @LINUX_HOST_TRUE@am__append_2 = -fPIC
+                @LINUX_HOST_TRUE@am__append_3 = -fPIC"""),
+            ""
+        )
+
+    def _configure_autotools(self):
+        if not self._env_build:
+            self._env_build = AutoToolsBuildEnvironment(self)
+            self._env_build.configure(configure_dir=self._source_subfolder)
+        return self._env_build
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version], destination=self._source_subfolder, strip_root=True)
+
+    def build(self):
+        if self.settings.os == "Linux":
+            self._patch_sources()
+
+        env_build = self._configure_autotools()
+        env_build.make()
+
+    def package(self):
+        self.copy("LICENSE", src=self._source_subfolder, dst="licenses")
+        env_build = self._configure_autotools()
+        env_build.install()
+        tools.rmdir(os.path.join(self.package_folder, "share"))
+        tools.rmdir(os.path.join(self.package_folder, "lib", "pkgconfig"))
+
+    def package_info( self ):
```

Comment:
Some help is needed here. `sentry-native` consumes this package using `pkgConfig`, but I'm not familiar with that system, so I don't know how to model it on Conan.
https://github.com/getsentry/sentry-native/blob/master/CMakeLists.txt#L439

Reply:
I'll fork this recipe and open a pr with some fixes.

Reply:
I think it would be interesting to create a `sentry-breakpad` recipe (such as I'm doing for `sentry-crashpad`), to see what components are needs to interface it with `sentry-native`.